src/DotNetWorker.OpenTelemetry/DotNetWorker.OpenTelemetry.csproj

```diff
@@ -0,0 +1,38 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
```

Comment:
Also you may need to include this in https://github.com/Azure/azure-functions-dotnet-worker/blob/2bd3b2de3f00291e03de19d59543cbc3a8fbc8a4/build/DotNetWorker.Core.slnf#L4 so that the build process produces a package for this project.